pages/common/jest.md

```diff
@@ -0,0 +1,23 @@
+# jest
+
+> Delightful JavaScript Testing.
+
+- Run all tests:
+
+    jest
+
+- Run tests filtered by spec name:
+
+    jest -t name-of-spec
+
+- Run tests relating to uncommitted files:
+
+    jest -o
+
+- Run using interactive watch mode to rerun tests for changed files:
```

Comment:
How about - "Automatically re-run tests on changes to files."

Reply:
That would be shorter, but I felt it was more descriptive to indicate that this command runs with a REPL. From `jest --help`:

> Watch files for changes and rerun tests related to changed files.

Reply:
I suggested this to match with other pages we have (webpack). Wherever possible, we strive to remain consistent throughout the pages. If we can tweak an example to be more lucid in its meaning, then we should use that by all means. If we just copy over the documentation, then it just becomes another man page copy :)

But if you want to keep that wording, I suggest - "Watch files for changes and re-run the tests in those files". Just avoided using "change" twice.